### PR TITLE
Add Axolotl v0.4.0 quickstart

### PR DESCRIPTION
**Pull request by qiyuangong**
## Description

Add GPU quick start for axolotl
https://qiyuan-ipex-llm-test.readthedocs.io/en/axolotqk/doc/LLM/Quickstart/axolotl_quickstart.html
### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [x] Document test
- [ ] ...

### CODE REVIEW

File: docs/readthedocs/source/doc/LLM/Quickstart/axolotl_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+The following example will introduce finetuning [Llama-2-7B](https://huggingface.co/meta-llama/Llama-2-7b) with [alpaca_2k_test](https://huggingface.co/datasets/mhenrichsen/alpaca_2k_test) dataset using LoRA and QLoRA.
+
+Note that you don't need to write any code in this example.
+
+| Model | Dataset | Finetune method |
+|-------|-------|-------|
+| Llama-2-7B | alpaca_2k_test | LoRA (Low-Rank Adaptation)  |
+| Llama-2-7B | alpaca_2k_test | QLoRA (Quantized Low-Rank Adaptation) |
+
+For more technical details, please refer to [Llama 2](https://arxiv.org/abs/2307.09288), [LoRA](https://arxiv.org/abs/2307.09288) and [QLoRA](https://arxiv.org/abs/2305.14314).
```
Comment (plusbang) [inline docs/readthedocs/source/doc/LLM/Quickstart/axolotl_quickstart.md:57]: The LoRA link is not correct.

File: docs/readthedocs/source/doc/LLM/Quickstart/axolotl_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+```cmd
+wget https://github.com/intel-analytics/ipex-llm/blob/main/python/llm/example/GPU/LLM-Finetuning/axolotl/lora.yml
+```
+
+If you use the model and dataset in local env, please modify the model path and dataset path in `lora.yml` to ensure Axolotl can access the offline model and data.
+
+```yaml
+# Please change to local path if model is offline
+base_model: /path/to/model/Llama-2-7b-hf
```
Comment (plusbang) [inline docs/readthedocs/source/doc/LLM/Quickstart/axolotl_quickstart.md:115]: should we just use model/dataset id in hf by default?

**Review by plusbang — APPROVED** [review]
Other LGTM

**Review by Oscilloscope98 — APPROVED** [review]

File: docs/readthedocs/source/doc/LLM/Quickstart/axolotl_quickstart.md (hunk 2 above)
Comment (qiyuangong) [reply docs/readthedocs/source/doc/LLM/Quickstart/axolotl_quickstart.md]: Yes. Change back to default and add comments

File: docs/readthedocs/source/doc/LLM/Quickstart/axolotl_quickstart.md (hunk 1 above)
Comment (qiyuangong) [reply docs/readthedocs/source/doc/LLM/Quickstart/axolotl_quickstart.md]: Addressed :)

**Pull request merged**